.gitignore

```diff
@@ -13,3 +13,6 @@ node_modules/
 openmrs.log
 .classpath
 .DS_Store
+/.apt_generated/
+/.apt_generated_tests/
+/bin/
```

Comment:
any reason for adding these files?

Reply:
Thanks, these files are very usefull while checking out bin folder and instantiating test classes. , scripts for selenium Mere removing of it result in to un wanted changes . So they are very helpful in UI tests.

Reply:
> Mere removing of it result in to un wanted changes 

are you sure we need these files?

Reply:
Resolved thanks.